tests/wpt/metadata/html/obsolete/requirements-for-implementations/other-elements-attributes-and-apis/document-color-02.html.ini

```diff
@@ -12,3 +12,6 @@
   [document: fg/bg/link/vlink/alink-color 5]
     expected: FAIL
 
+  [document: fg/bg/link/vlink/alink-color]
```

Comment:
These now fail because both `vlink` and `text` attributes were not implemented, so `undefined === undefined`. Now that `text` is implemented, they no longer are equal since `"" !== undefined`